src/strategies/frax-finance/index.ts

```diff
@@ -1,3 +1,5 @@
+// fix issue with veFXS, rolled back to commit 6ebe51fa883e13a158516d36959478fbcf0c1a82
```

Comment:
@jasonhuan what is this pr for? make sure to push all changes 🤔 .. all I see is this comment in file changes

Reply:
ah sorry, should be pushed now